docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2.1'
+
+services:
+
+  base-etl:
+    build:
+      context: .
+    volumes:
+      - .:/code
+    environment:
+      AZURE_STORAGE_CONNECTION_STRING: '${AZURE_STORAGE_CONNECTION_STRING}'
+      CONTAINER_NAME: ${CONTAINER_NAME}
+      SYNDICATION_API_KEY: ${SYNDICATION_API_KEY}
+      API_URL: http://v1.syndication.nhschoices.nhs.uk/services/types/sexualhealthinformationandsupport
+      OUTPUT_FILE: dev-shis-data
```

Comment:
Can this be updated to `${OUTPUT_FILE-dev-shis-data}` so the file name can be overwritten locally please?